src/Microsoft.Framework.Runtime.Hosting/Json/JsonDeserializerException.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+namespace Microsoft.Framework.Runtime.Json
+{
+    internal class JsonDeserializerException : Exception
```

Comment:
Are you sure this is supposed to be internal? How is someone going to catch this?